app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/bitcoind/BlockchainResult.scala

```diff
@@ -52,7 +52,23 @@ case class GetBlockWithTransactionsResult(
     nextblockhash: Option[DoubleSha256DigestBE])
     extends BlockchainResult
 
-case class GetBlockChainInfoResult(
+sealed trait GetBlockChainInfoResult extends BlockchainResult {
+  def chain: NetworkParameters
+  def blocks: Int
+  def headers: Int
+  def bestblockhash: DoubleSha256DigestBE
+  def difficulty: BigDecimal
+  def mediantime: Int
+  def verificationprogress: BigDecimal
+  def initialblockdownload: Boolean
+  def chainwork: String // How should this be handled?
```

Comment:
nit: maybe now a good time to change this to a big int

Reply:
So this is unrelated to db stuff that's been going on and is about the fact that the bitcoind rpc returns a string with quotes around it (presumably to maintain precision). This can either be handled by changing the `Reads[BigInt]` to accept either a number or a `String` or to do something custom with just this `case class`, either way I think it would be better in a different PR